signalfx-tracing/signalfx-python-tracing/django/requirements.txt

```diff
@@ -0,0 +1,6 @@
+django
+djangorestframework
+coreapi
+pyyaml
```

Comment:
I'm not seeing coreapi or pyyaml used anywhere

Reply:
localhost:8001/schema